conda/auxlib/__init__.py

```diff
@@ -76,6 +76,9 @@ def __eq__(self, other):
     def __hash__(self):
         return hash(_Null)
 
+    def __str__(self):
```

Comment:
This is just a minor style fix so the logs show `Null` instead of the long `<Null object at 0x123456>` you get by default. The point is to mimic `None`.